model/common/README.md

```diff
@@ -14,32 +14,34 @@ Check the `README.md` at the root of the `model` folder for installation instruc
 
 module containing IO funcitonality for ICON4Py.
 
-The IO module requires the installation of the `io` optional dependencies defined in [pyproject.toml](../../../pyproject.toml)
+The IO module requires the installation of the `io` optional dependencies defined in [pyproject.toml](./pyproject.toml)
 of the `icon4py-common` package and can be installed with the following command:
 
 ```bash
-pip install icon4py-common[io]
+uv pip install .[io]
 ```
 
-or by using
+or even better by running:
 
 ```bash
-pip install -r requirements-dev.txt
+uv sync --extra io  # or `uv sync --extra all` which includes everything
 ```
 
-which installs all `ICON4Py` packages including the IO dependencies.
+at the top-level folder of the repository, which installs all `ICON4Py` packages including the IO dependencies.
 
-### decomposition
+### Distributed run
 
 Contains infrastructure for parallel implementation of `icon4py/model`.
```

Comment:
That sentence does not make a lot of sense. I assume the title `decomposition`was referring to the package. 
```suggestion
The package `decomposition` contains infrastructure for parallel implementation of `icon4py/model`.
```